extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisInputDStream.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.spark.streaming.scheduler.ReceivedBlockInfo
 import org.apache.spark.streaming.{Duration, StreamingContext, Time}
 
 private[kinesis] class KinesisInputDStream[T: ClassTag](
-    @transient _ssc: StreamingContext,
+    _ssc: StreamingContext,
```

Comment:
Note that this matches the change that was made to FlumePollingInputDStream: https://github.com/apache/spark/commit/c1bc4f439f54625c01a585691e5293cd9961eb0c#diff-0a2347be859b78b562e3e8e8e17a1f1fL49